Language/Expressions/Await_Expressions/syntax_t03.dart

```diff
@@ -9,9 +9,7 @@
 /// @description Check that it is a compile error if unaryExpression is missing
 /// @author a.semenov@unipro.ru
 
-
 import 'dart:async';
-import '../../../Utils/expect.dart';
 
 f() {
```

Comment:
Unused function? (also a few other tests like it)

Reply:
Yes. Thank you. Unused code removed